util/config/run-in-venv-with-python-bindings.bash

```diff
@@ -17,4 +17,8 @@ if [ ! -d "$chpl_frontend_py_deps/chapel" ]; then
   exit 1
 fi
 
+if [ "$CHPL_SANITIZE" == "address" ]; then
+  export LD_PRELOAD=$(gcc -print-file-name=libasan.so)
```

Comment:
Instead of `gcc`, this should use `CHPL_HOST_CC`. I think the best way to get this in bash is probably `printchplenv --value --only CHPL_HPST_CC` 

Also note that with clang the file you need is `libclang_rt.asan.so`